framework/include/kernels/ConservativeAdvection.h

```diff
@@ -28,12 +28,25 @@ class ConservativeAdvectionTempl : public GenericKernel<is_ad>
 protected:
   virtual GenericReal<is_ad> computeQpResidual() override;
   virtual Real computeQpJacobian() override;
+  virtual Real computeQpOffDiagJacobian(unsigned int jvar) override;
   virtual void computeResidual() override;
   virtual void computeJacobian() override;
 
+  /// Scalar material property
```

Comment:
More information about what role this plays in the advection term please. Is it the quantity advected by the velocity? A multiplying coefficient?